steps/discover-repos.rb

```diff
@@ -97,44 +82,110 @@ def mock_reps(page, size, licenses)
   }
 end
 
-def cooldown(opts, found)
-  puts "Let's sleep for #{opts[:pause]} seconds to cool off GitHub API \
-(already found #{found.count} repos, need #{opts[:total]})..."
-  sleep opts[:pause]
+def process_year(year, github, context)
+  query = build_query(year, context[:opts])
+  puts "Querying for repositories created in #{year}..."
+
```

Comment:
@lueFlake what's the point of this empty line? https://www.yegor256.com/2014/11/03/empty-line-code-smell.html